### DISCUSSION TITLE
launch channel one after one with pause

### DISCUSSION THREAD

**Discussion by postheimer** [nextgenhealthcare/connect#4707] [Q&A]
Hello , 

I search a solution in my problem

i have 3 channel , every channel make a list of process and make variable for the other channel ...

but i want make a scenario , one channel run and wait 30 sec and launch the second  ... 30 sec ... and launch the last

a possibility by postprocessor ?

**Comment by tonygermano** [comment]
What you likely want to do is go to the Summary tab of the middle channel, then click the **Set Dependencies** button. From there click the **Deploy/Start Dependencies** tab. Here you can set that this channel depends on the first channel and is depended on by the third channel.

Assuming you do all of your initialization in the deploy script, mirth will handle ensuring that one channel has completely deployed before deploying the next one.

**Reply by postheimer** [reply]
thanks for your reply , but in this method the second channel was run too fast

maybe in deploy script he as a method for pause the first run ?

**Reply by pacmano1** [reply]
I'm curious, why can't those three channels be consolidated into one?   Also you can always start all of them at the same time and have them not poll on start and spread the poll time out appropriately.

**Reply by postheimer** [reply]
:-) , the first copy a list of jpg and extract the patient mrn from the filename and copy the jpg on server
the second extract from a db2 the information of the database dob, name ... and make a pdf to a share
the 3 is create a linker file with information of the first and second to a database  and the link of the pdf ...

i know is strange but the infra is very slow and the copy of jpg take time , and the pdf of the second is too fast ...

**Reply by pacmano1** [reply]
That seems to be implementable in a single channel with multiple destinations.

**Reply by postheimer** [reply]
i have re work my channel 

i have reduce to 2 channel but i need always the second channel pause 15 sec before run 

i have search with channelutil pause but no found exemple ...
